src/libraries/SwapMath.sol

```diff
@@ -102,7 +102,8 @@ library SwapMath {
                 if (protocolFeePips > 0) {
```

Comment:
if `exactIn && sqrtRatioNextX96 != sqrtRatioTargetX96`, the whole fee is going to LP and nothing to protocol, which is why some tests are failing. How should protocol fee be calculated here?